test/threadstest.c

```diff
@@ -1223,8 +1225,10 @@ int setup_tests(void)
 #if defined(OPENSSL_THREADS)
     ADD_TEST(torture_rw_low);
     ADD_TEST(torture_rw_high);
+#ifndef OPENSSL_SYS_MACOSX
```

Comment:
Nit: space after `#` - applies to all other places where `#ifdef/#endif` was added